Hmm should have access but in any case here's the error:

```sh
##[error]./pandas/core/internals/managers.py(556,48): error E128: continuation line under-indented for visual indent
```

Also if you merge master instead of rebasing shouldn't need to force push which is preferable with maintaining GitHub comment history

--- a/pandas/core/internals/managers.py
+++ b/pandas/core/internals/managers.py
@@ -553,7 +553,7 @@ class BlockManager(PandasObject):
                 return isna(values)
             if hasattr(s, 'asm8'):
                 return _compare_or_regex_search(maybe_convert_objects(values),
-                                               getattr(s, 'asm8'), regex)
+                                                getattr(s, 'asm8'), regex)
             return _compare_or_regex_search(values, s, regex)
 
         masks = [comp(s, regex) for i, s in enumerate(src_list)]

